cert-manager/cluster-issuer.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+  namespace: kube-system
+spec:
+  acme:
+    # The ACME server URL
+    server: https://acme-v02.api.letsencrypt.org/directory
+    # Email address used for ACME registration
+    email: cloud-deployment@forgerock.com
```

Comment:
just checking - this goes to a real group, right?

Reply:
well, this did exist but I've tested it and haven't had a response. Have you received anything?